run_dev_node.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Start Nitro dev node in the background
+echo "Starting Nitro dev node..."
+docker run --rm --name nitro-dev -p 8547:8547 offchainlabs/nitro-node:v3.2.1-d81324d --dev --http.addr 0.0.0.0 &
+
+# Wait for the node to initialize
+echo "Waiting for the Nitro node to initialize..."
+sleep 15  # Adjust the sleep time if necessary based on node startup time
```

Comment:
instead of a fixed delay, waiting until Nitro node is running seems better:
```
until [[ "$(curl -s -X POST -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"net_version","params":[],"id":1}' http://127.0.0.1:8547)" == *"result"* ]]; do
    sleep 0.1;
done;

```